contrib/completion/bash/docker

```diff
@@ -798,7 +802,7 @@ _docker_run() {
 			__docker_capabilities
 			return
 			;;
-		--cidfile|--env-file)
+		--cidfile|--cgroup-parent|--env-file|--label-file)
```

Comment:
If you have to touch this file for some other reason, then please also place cgroup-parent alphabetically sorted.